machine_configuration/machine-config-pin-preload-images-about.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="machine-config-pin-preload-images-about"]
+= About pinning and preloading images
```

Comment:
```suggestion
= About preloading and pinning images
```
Changed to match title in topic map.

Reply:
I went with pin and preload as the RFE, Jira issues, etc. used that format. However, technically, pre-loading does happen before pinning. I've been going back and forth over which way to present it (thus the different topic map entry, good catch!). I will ask about how we want to name this feature.